http/file_server_test.ts

```diff
@@ -57,7 +58,7 @@ async function startFileServer({
     ],
     cwd: moduleDir,
     stdout: "piped",
-    stderr: "null",
+    stderr: "inherit",
   });
```

Comment:
Adding this line will make debugging easier as error messages will be displayed instead of discarded. It was a minor change so I've included it in this PR.